scripts/observer/index.tsx

```diff
@@ -1,5 +1,13 @@
 import React from 'react';
 import {Box, Text, Color, render} from 'ink';
+import {getGitStagedFiles, getDependencies} from '../../treebuilder';
+
+(async function() {
+  const stagedFiles = await getGitStagedFiles('src/');
+  console.log(stagedFiles);
+  const data = await getDependencies(/* how do I use this function? */);
```

Comment:
@amrocha I couldn't figure out what was expected to be passed here. A little nudge would be appreciated!